assets/js/components/settings/settings-module.js

```diff
@@ -275,7 +276,7 @@ class SettingsModule extends Component {
 											googlesitekit-heading-4
 											googlesitekit-settings-module__title
 										">
-											{ moduleIcon( slug, false, '24', '26', 'googlesitekit-settings-module__title-icon' ) }
+											<ModuleIcon slug={ slug } width={ 24 } height={ 26 } className="googlesitekit-settings-module__title-icon" />
```

Comment:
Same question as before (also applies to several other locations): Shouldn't the dimensions for module icons always be square?

Reply:
@felixarntz Answered this in a previous comment but figured I'd add thoughts here as well. I'm curious if setting non-square values is a leftover from when the icons were images or having the assumption that SVGs scale the same way that images do. As all of the icons are rectangles, setting them with equal width/height values (if they were images) would result in visual distortion as the images would scale up to fill the space whereas SVGs would not scale up the smaller value. I tried to find the commit where these values were added but they point to the initial commit. I also tried to find discussion about the topic by searching the repository for `moduleIcon` but it doesn't return any results.